test/integration/testdata/gen-plugin-e2e-configmap.golden

```diff
@@ -24,21 +20,22 @@ spec:
   command:
   - /run_e2e.sh
   env:
+  - name: E2E_EXTRA_ARGS
+    value: --progress-report-url=http://localhost:8099/progress
   - name: E2E_FOCUS
     value: \[Conformance\]
-  - name: E2E_SKIP
-    value: \[Disruptive\]|NoExecuteTaintManager
   - name: E2E_PARALLEL
     value: "false"
+  - name: E2E_SKIP
+    value: \[Disruptive\]|NoExecuteTaintManager
   - name: E2E_USE_GO_RUNNER
     value: "true"
-  - name: E2E_EXTRA_ARGS
-    value: --progress-report-url=http://localhost:8099/progress
+  - name: SONOBUOY_K8S_VERSION
+    value: v123.456.789
```

Comment:
This value should have been added but it never was part of the unique `gen plugin e2e` flow